ttsai/rotator_verification.ipynb

```diff
@@ -0,0 +1,3831 @@
+{
```

Comment:
<p>How are flags generally tested?</p><p>How are the following flags are checked during the tests?</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;pt2ptMoveComplete</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;newPt2PtCommand</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;moveSuccess</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;positionSet</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;initializationComplete</p><p>&nbsp;</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;stopComplete</p><p>&nbsp;</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;followingError</p><p>&nbsp;</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;positionFeedbackFault</p><p>&nbsp;</p><p>-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;rotatorOdometry_deg</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/72/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>Some of them (such as positionSet) can not be tested because there is no related telemetry. But you can base on move() to see the result.</p><p> </p><p>Some of them (such as pt2ptMoveComplete) is inherited in the events I subscribed from EFD (if you see the implementation of ts_mtrotator, you will see the way to combine several telemetry into a single event). But the straight forward way is to generate the binary file from EUI and analyze it.</p><p> </p><p>But I think the system engineer team prefers to use the EFD.</p>

Reply:
<p>Please add some text showing the mapping of internal flags to the events in the EFD</p>

Reply:
Updated in the commit: 137c4fd to f4b262b already. There is the case-sensitive error in your folder and I could not reach the LSP at summit and Tucson Teststand to rewrite the history. Stay like this at this time. Thanks!